website/docs/plugins/ray_launcher.md

```diff
@@ -64,47 +66,42 @@ defaults:
 config your AWS credentials first, tutorials can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
 
 `ray autoscaler` expects a yaml file to provide specs for the new cluster, which we've schematized in `hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.conf.__init__.RayClusterConf`, 
-we provide a sample yaml file in `conf/hydra/launcher/ray_aws.yaml`. You can either edit the yaml file or override it from the command line to override the 
-default values.
+we provide a sample yaml file in `conf/hydra/launcher/ray_aws.yaml`. You can override the default values in your app config or from command line.
 
-We use `cloudpickle` behind the scene to launch jobs on `ray`, because of the way `cloudpickle` works, we expect your 
-project to be structured in a particular way: your project must either be a single module or a pip installed package.
-For example, the following structure would not work:
-```
-bad_structure
-├── a.py  # uses b.py objects in a.py
-└── b.py
-```
-If you need to separate your code into distinct modules, please organize your project as a package and `pip install` it.
 
 <div class="alert alert--info" role="alert">
 NOTE: To make sure plugin runs as expected, you need to make sure your local machine and your AWS cluster runs the same version of: `ray`, `hydra-core` and `python`. You can 
 easily install the correct software version by overriding `RayClusterConf.setup_commands`
 </div><br/>
 
-Now we can go ahead and run `my_app.py` using `ray_aws` launcher
+Now we can go ahead and run `train.py` using `ray_aws` launcher
```

Comment:
Instead of showing the example with ray_aws being in the defaults, show it being overridden from the command line.
$ python train.py hydra/launcher=ray_aws.

this works more naturally to switch to the local launcher, which should also be mentioned in the readme.